src/main/scala/scalismo/faces/parameters/RenderObject.scala

```diff
@@ -141,6 +143,20 @@ object MoMoInstance {
            modelURI: URI): MoMoInstance = {
     fromCoefficients(MoMoCoefficients.zeros(shapeComponents, colorComponents, expressionComponents), modelURI)
   }
+
+  /** create a random coefficient vector with its elements N(0,1) distributed. */
+  def sample(model: MoMo, modelURI: URI)(implicit rnd: Random): MoMoInstance = {
+    val fullShapeCoeffs = for (_ <- 0 until model.neutralModel.shape.rank) yield Gaussian(0, 1).draw()
```

Comment:
Why creating a Gaussian(0,1) three times? You could also use rng.breezeRandBasis.gaussian.draw() instead.